src/routes/Detail.js

```diff
@@ -111,6 +120,13 @@ const Detail = ({ loadApi, detail }) => {
                   deepLinking
                   docExpansion="list"
                   spec={detail.spec}
+                  requestInterceptor={async (req) => {
```

Comment:
Can you define the interceptor with the `useCallback` somewhere in the component? Better than recreating it on each render.